### ISSUE TITLE
First item is illuminated without selection.

### ISSUE THREAD

**Issue by micahilbery** [elementary/files#552]
Files version: 0.3.6
elementary OS Version: Juno Beta 1

By default when I open files or a directory in files, the first item displayed is always "illuminated" despite not ever being selected or hovered over. This is similar to what's reported in #535 but different in that no selection has to be made.

![screenshot from 2018-07-17 09 56 18](https://user-images.githubusercontent.com/10608836/42825771-5cd60998-89a8-11e8-80d8-705f7e3bd3f7.png)
![screenshot from 2018-07-17 09 59 10](https://user-images.githubusercontent.com/10608836/42825775-5f851242-89a8-11e8-837c-7b473e096bce.png)

**Comment by jeremypw** [comment]
I do not think this is a bug - the highlighting is showing the "cursor" position.  Pressing an arrow key will select the highlighted position. Mostly the item at the cursor position is already selected which obscures the highlighting.  However if you select any item and the deselect using the keyboard or helper it you will see the same highlighting.

**Comment by jeremypw** [comment]
I guess it could be considered an issue that both the cursor and hover position are shown by the same highlighting but I am not sure what the downside of that is.

**Comment by micahilbery** [comment]
Ok, I could see that. For me personally it's just kinda annoying while working on icons. It illuminates the icon I'm trying to view as it will be rendered by the system. The illumination makes it not show accurately and if it's the only file in the directory there's no way to show it without the illumination without creating another file or folder.

**Comment by jeremypw** [comment]
Oh I see. that's not a use case I considered.  I guess if the cursor position were shown some other way, e.g. outlining the name, without highlighting the icon then it would solve your issue.  A workaround for now is to press tab which moves keyboard focus out of the view (Shift-Tab to move it back). Or click on the searchbar for the same effect.

**Comment by jeremypw** [comment]
Thanks for your efforts @lainsce !  I'll bring it to the attention of the UX team, who are the final arbiters of this sort of thing.  There are unfortunately technical difficulties in styling GtkCellRenderers as they do not respond to CSS styling like widgets but we will do our best.

**Comment by jeremypw** [comment]
Ah - I see you have already submitted to Slack.

**Comment by rugglcon** [comment]
@jeremypw I think in any case, the default behavior would just be that no folder is highlighted unless you've clicked on it. Every time I open Files I see the first item in the list with a brighter colored icon than the rest and it makes me look at that item differently until I realize that it's not actually different. I think more input from a few people would be good to maybe get their viewpoint and what their workflow is to see what a consensus says. If it were me making the decision, I would want it to go back to the old behavior of not highlighting anything on startup.

FYI I'm on eOS 5, downloaded on 10/16/18 and installed today 10/17/18.

**Comment by jeremypw** [comment]
The problem is that the icon is highlighted to show cursor position as well as when hovered or selected.  An attempt was made to resolve this in #579 but it was unsuccessful.  Will probably have another attempt at some point.  We need to show cursor position for keyboard users.

**Comment by jeremypw** [comment]
@eljefuri : How would you deal with the cursor position issue?

**Comment by eljefuri** [comment]
@jeremypw I'm studying what you posted. I think my primary issue with the solution proposed is that I am not in agreement that Files should share the single-click-to-open aspect of the rest of the OS, primarily because this forces the existence of what are for me terribly confusing modes. I need a little bit more time with it to think this through.

**Comment by andreasfelix** [comment]
In my opinion the cursor position should only be shown after an <kbd>arrow key</kbd> was pressed. Pressing <kbd>Esc</kbd> or clicking on an empty area should hide the cursor position and reset it to the first item.

Also in my opinion the cursor position should not be indicated by highlighting the icon, instead i would prefer a different background color (similar to lainsce's mockup).

**Comment by lainsce** [comment]
@jeremypw Any attempts on alleviating this with the colored background(can be the GTK theme accent color) instead of the highlight? :joy:

**Comment by jeremypw** [comment]
There are already 11 unmerged PRs going back to last November so I am not inclined to add to queue unless it is a fairly urgent issue :disappointed:

**Comment by lainsce** [comment]
I think it's best to make a PR irregardless of a queue since it'll be reviewed and merged and will help the UX of all users of the OS :thinking:

**Comment by jeremypw** [comment]
The problem is I am loathe to produce a PR until it is decided how to and when to show the keyboard focus position.

**Comment by jeremypw** [comment]
We should revisit this in the Gtk4 version as the views will need complete rewriting anyway and there will be more scope for different appearances because widgets will be used.
